### DISCUSSION TITLE
Chat history store

### DISCUSSION THREAD

**Discussion by MikitaPetukhou** [open-webui/open-webui#6948] [General]
Hi Community!

I'm new for the OpenWebUI, just have a few questions if anyone can help with?
- ~~where the chat history is stored along with necessary metadata (eg. about user and sessions)?~~
- is it possible to connect a NoSql database for that?

It would be great if anyone can guide on the topic.

**Comment by MikitaPetukhou** [comment]
ah.. ok.. I have found an sqlite db in app/backend/data/webui.db and was able to explore it from VsCode SQLite extension.
I will leave it here. maybe someone will find it useful. 

No the remaining question is about NoSQL db

**Reply by arcovp** [reply]
tailing this

**Reply by Fade78** [reply]
There is a documentation about that there: [https://docs.openwebui.com/tutorials/tips/sqlite-database](https://docs.openwebui.com/tutorials/tips/sqlite-database)
